repairnator/src/main/java/fr/inria/spirals/repairnator/process/ProjectScanner.java

```diff
@@ -190,14 +195,14 @@ public int getTotalScannedBuilds() {
         return result;
     }
 
-    private boolean testBuild(Build build) {
+    private boolean testBuild(Build build, BuildStatus targetBuildStatus) {
         Repository repo = build.getRepository();
```

Comment:
In fact you're using the targetBuildStatus as a boolean value: so maybe the code will be more clear if you pass a boolean as an argument named "targetFailing"